### ISSUE TITLE
[Bug] Documentation won't load in Pybricks Beta

### ISSUE THREAD

**Issue by laurensvalk** [pybricks/support#896]
This seems to happen only in my offline version of Pybricks Beta. If I open it in a new profile, the docs do load.

A hard refresh does not seem to help.

![image](https://user-images.githubusercontent.com/12326241/210234852-743a0a48-379d-4f6b-981e-ee2b65c4ff0c.png)

**Comment by dlech** [comment]
In the chrome dev tools, go to the *Application* tab, select *Service Workers* and click *Unregister*, then refresh the page. 

![image](https://user-images.githubusercontent.com/963645/210260858-dd136b31-d817-46e1-8331-83ab3c0f0b5e.png)

There was an older release that had a bug like this, so if you were upgrading from that version, it could explain the issue.

**Comment by dlech** [comment]
Part of the problem is that chrome doesn't clear the cache for the iframe when you do <kbd>ctrl</kbd>+<kbd>r</kbd>.

**Comment by dlech** [comment]
is this still a problem?

**Comment by laurensvalk** [comment]
Yes. But I haven't cleared the cache yet as I don't want to manually re-upload a hundred programs.

I was just going to wait until the next release to see if that refreshes the docs.

**Comment by dlech** [comment]
There is nothing a new release can do to fix the problem since the bug was caused by a previous release. Clearing the cache and unregistering the app does not clear the indexeddb where the files are stored.
